src/commands/sites/sites-create-template.js

```diff
@@ -0,0 +1,251 @@
+// @ts-check
+
+const slugify = require('@sindresorhus/slugify')
+const inquirer = require('inquirer')
+const pick = require('lodash/pick')
+const sample = require('lodash/sample')
+const fetch = require('node-fetch')
+const prettyjson = require('prettyjson')
+const { v4: uuidv4 } = require('uuid')
+
+const { chalk, error, log, logJson, track, warn } = require('../../utils')
+const { getGitHubToken } = require('../../utils/gh-auth')
+
+const SITE_NAME_SUGGESTION_SUFFIX_LENGTH = 5
+
+let ghToken
+
+// Templates are hardcoded for now
+const templates = [
+  {
+    name: 'Next.js Blog theme',
+    sourceCodeUrl: 'https://github.com/netlify-templates/nextjs-blog-theme',
+    slug: 'netlify-templates/nextjs-blog-theme',
+  },
+  {
+    name: 'Next.js Starter',
+    sourceCodeUrl: 'https://github.com/netlify-templates/next-netlify-starter',
+    slug: 'netlify-templates/next-netlify-starter',
+  },
+  {
+    name: 'Nuxt.js Starter',
+    sourceCodeUrl: 'https://github.com/Gomah/bluise',
+    slug: 'Gomah/bluise',
+  },
+  {
+    name: 'Hugo Blog',
+    sourceCodeUrl: 'https://github.com/netlify-templates/one-click-hugo-cms',
+    slug: 'netlify-templates/one-click-hugo-cms',
+  },
+]
+
+/**
+ * The sites:create-template command
+ * @param {import('commander').OptionValues} options
+ * @param {import('../base-command').BaseCommand} command
+ */
+const sitesCreate = async (options, command) => {
+  const { api } = command.netlify
+
+  await command.authenticate()
+
+  console.log(
+    `Choose one of our starter templates. Netlify will create a new repo for this template in your GitHub account.`,
+  )
+
+  const templateUrl = await inquirer.prompt([
```

Comment:
what about some flags to skip inquirer in environments where no tty is available?

Like `--template-url` to fetch the template from (then you always have the possibility to use different templates that are not in this list.

Reply:
I made some changes to implement this, let me know what you think!